src/imputation/tmi_imputation.py

```diff
@@ -439,9 +459,11 @@ def run_tmi(full_df, target_variables, sic_mapper):
 
     df = pd.concat([final_tmi_df, shortform_df])
 
-    final_df = df.sort_values(
-        ["reference", "instance"], ascending=[True, True]
-    ).reset_index(drop=True)
+    df = df.sort_values(["reference", "instance"],
+                        ascending=[True, True]).reset_index(drop=True)
```

Comment:
the formatting in the deleted code is correct:
df = df.sort_values(
      etc
).etc

Reply:
Changed